inode/inode.go

```diff
@@ -351,13 +352,78 @@ func (vS *volumeStruct) inodeCacheTouch(inode *inMemoryInodeStruct) {
 	vS.Unlock()
 }
 
+// TODO: What happens if the volume is removed?  What if
+// failover?
+func (vS *volumeStruct) inodeCacheDiscard() {
+	inodesToDrop := uint64(0)
+
+	vS.Lock()
+
+	if (vS.inodeCacheLRUItems * globals.inodeSize) > vS.inodeCacheLRUMaxBytes {
+		// Check, at most, 1.25 * (minimum_number_to_drop)
```

Comment:
Unless you always evict the total # of inodes in excess of the limit at each timer tick, it is quite possible that a too large timer tick interval may fall further and further behind... hence one of a few reasons why I prefer a "triggered-by-exceeding-the-limit" mechanism over a "polling" one.